WordPress/src/main/java/org/wordpress/android/datasets/ReaderPostTable.java

```diff
@@ -538,20 +549,30 @@ public static int removeTagsFromPost(long blogId, long postId, final ReaderTagTy
         }
 
         String[] args = {Integer.toString(tagType.toInt()), Long.toString(blogId), Long.toString(postId)};
-        return ReaderDatabase.getWritableDb().delete(
+        int rowsDeleted = ReaderDatabase.getWritableDb().delete(
                 "tbl_posts",
                 "tag_type=? AND blog_id=? AND post_id=?",
                 args);
+
+        if (rowsDeleted > 0) {
+            EventBus.getDefault().post(ReaderPostTableActionEnded.INSTANCE);
+        }
+        return rowsDeleted;
     }
 
     public static int deletePostsInBlog(long blogId) {
         String[] args = {Long.toString(blogId)};
-        return ReaderDatabase.getWritableDb().delete("tbl_posts", "blog_id = ?", args);
+        int rowsDeleted = ReaderDatabase.getWritableDb().delete("tbl_posts", "blog_id = ?", args);
```

Comment:
Another instance of the method directly returning the int value. I changed to keep the value local and will only post the new event is rows have actually been deleted.